server/middlewares/postValidator.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Validates a post object
+ * @param {object} req
+ * @param {object} res
+ * @param {object} next
+ */
+const postValidator = (req, res, next) => {
+  const stringPattern = /([a-zA-Z])+/;
```

Comment:
The use of Validator Js would have saved some space here, the package is already installed just import and use.

Reply:
@davidokonji I didn't see a method to both validate a string and invalidate an empty string on their documentation. So I opted for regex. 

Joi Joi, oh how badly we need you.